docs/resources/permissions.md

```diff
@@ -151,6 +151,10 @@ resource "databricks_group" "eng" {
     display_name = "Engineering"
 }
 
+resource "databricks_service_principal" "main" {
+  display_name = "main"
```

Comment:
Please also explicitly mention for which cloud is it. The API is rather confusing, unfortunately.

Reply:
I did it by changing the name of the "databricks service principal".

Reply:
Great!